.jenkinsci-new/helpers/remove-function-lines.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+
+# https://github.com/JeremyAgost/lcov-llvm-function-mishit-filter
+# https://github.com/linux-test-project/lcov/issues/30
+
+import argparse
+import subprocess
+
+def demangle(symbol):
+    p = subprocess.Popen(['c++filt','-n'], stdin=subprocess.PIPE, stdout=subprocess.PIPE)
```

Comment:
Coverage almost always runs at `gcc5`, but it possible to run coverage at any other compiler
with **`Custom command`**:
ex. of parameters to run build
`custom_cmd: x64linux_compiler_list = ['clang7']; coverage = true`
or on mac:
`custom_cmd: x64linux_compiler_list = []; mac_compiler_list = ['appleclang']; coverage = true`
This python code looks simple and hope it won't fail on mac, also until now nobody used coverage at mac